vimiv/config/configfile.py

```diff
@@ -6,6 +6,8 @@
 
 """Functions to read configurations from config file and update settings."""
 
+import re
+
```

Comment:
The import order is slightly funny here, I would order them as:
```
import configparser
import re
```

Reply:
Oh sure, somehow thought `configparser` is a local module :see_no_evil: